Cartridges/int_checkoutcom_sfra/cartridge/client/default/js/product/detail.js

```diff
@@ -1,33 +1,139 @@
 'use strict';
```

Comment:
i don't think we should add this file to our integration. Please can you look into removing this file also. we also don't need the product folder.